mrblib/itamae/resource_executor/group.rb

```diff
@@ -0,0 +1,42 @@
+module Itamae
+  module ResourceExecutor
+    class Group < Base
+      def apply(current, desired)
+        if desired.exist
+          if exist?(desired.groupname)
+            if desired.gid && desired.gid != current.gid
+              run_specinfra(:update_group_gid, desired.groupname, desired.gid)
```

Comment:
Why didn't you call `updated!` here? https://github.com/itamae-kitchen/itamae/blob/v1.9.9/lib/itamae/resource/group.rb#L22